.rubocop.yml

```diff
@@ -10,6 +10,9 @@ AllCops:
     - 'tmp/**/*'
   TargetRubyVersion: 2.0
 
+Rails/Date:
```

Comment:
Don't think this is needed in our own `.rubocop.yml`. 🤔

Reply:
Indeed.

Reply:
Oh... I'd had to disable it for my own project's CI. Did it in the wrong window. 😬